easybuild/tools/toolchain/toolchain.py

```diff
@@ -1049,39 +1060,70 @@ def handle_sysroot(self):
                 setvar('PKG_CONFIG_PATH', os.pathsep.join(pkg_config_path))
 
     def _add_dependency_variables(self, names=None, cpp=None, ld=None):
-        """ Add LDFLAGS and CPPFLAGS to the self.variables based on the dependencies
-            names should be a list of strings containing the name of the dependency
         """
-        cpp_paths = ['include']
-        ld_paths = ['lib64', 'lib']
-
-        if cpp is not None:
-            for p in cpp:
-                if p not in cpp_paths:
-                    cpp_paths.append(p)
-        if ld is not None:
-            for p in ld:
-                if p not in ld_paths:
-                    ld_paths.append(p)
-
-        if not names:
-            deps = self.dependencies
-        else:
-            deps = [{'name': name} for name in names if name is not None]
+        Add linker and preprocessor paths to dependencies to self.variables
```

Comment:
This docsrtring is not a repetition, this method adds paths to `self.variables`, while the other code results in this paths being added to `self.vars`. I know it is confusing, but I will not get into the `self.vars` vs `self.variables` situation in `Toolchain`. So that's why I added this docstring.